tests/test_constraints.py

```diff
@@ -1,13 +1,9 @@
 """
 Tests the constraints invariants
 """
-from __future__ import division
-from __future__ import print_function
-from __future__ import unicode_literals
 
 import unittest
-
-import ga4gh.common.utils as utils
+import utils as utils
```

Comment:
this must have been written by the same guy as last time. 
utils as utils...hmm